src/workerd/jsg/setup.h

```diff
@@ -60,7 +60,7 @@ class V8System {
 
   ~V8System() noexcept(false);
 
-  typedef void FatalErrorCallback(kj::StringPtr location, kj::StringPtr message);
+  typedef void FatalErrorCallback(kj::StringPtr location, kj::StringPtr messagex);
```

Comment:
```suggestion
  typedef void FatalErrorCallback(kj::StringPtr location, kj::StringPtr message);
```

Is the addition of the `x` intentional?

Reply:
I'll change it back ~to Twitter~